<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../tano/resources/logo/logo.qrc">
    <normaloff>:/logo/48x48/logo.png</normaloff>:/logo/48x48/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="FancyTabWidget" name="tabs" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRecorder">
    <property name="title">
     <string>&amp;Recorder</string>
    </property>
    <addaction name="actionRecorder"/>
    <addaction name="separator"/>
    <addaction name="actionRecord"/>
    <addaction name="separator"/>
    <addaction name="actionRecordNow"/>
    <addaction name="actionSnapshot"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionEditChannels"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionUpdate"/>
    <addaction name="actionSupport"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Vie&amp;w</string>
    </property>
    <addaction name="actionTop"/>
    <addaction name="actionLite"/>
    <addaction name="actionFullscreen"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAudio">
    <property name="title">
     <string>&amp;Audio</string>
    </property>
    <addaction name="actionMute"/>
    <addaction name="actionVolumeUp"/>
    <addaction name="actionVolumeDown"/>
   </widget>
   <widget class="QMenu" name="menuMedia">
    <property name="title">
     <string>&amp;Playback</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionBack"/>
    <addaction name="actionNext"/>
    <addaction name="actionTeletext"/>
    <addaction name="actionPreview"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>&amp;Video</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenUrl"/>
    <addaction name="separator"/>
    <addaction name="actionSchedule"/>
    <addaction name="actionScheduleCurrent"/>
    <addaction name="separator"/>
    <addaction name="actionTray"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuMedia"/>
   <addaction name="menuAudio"/>
   <addaction name="menuVideo"/>
   <addaction name="menuRecorder"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../tano/resources/logo/logo.qrc">
     <normaloff>:/logo/48x48/logo.png</normaloff>:/logo/48x48/logo.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="preferences-desktop">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="view-fullscreen">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open &amp;File</string>
   </property>
  </action>
  <action name="actionOpenUrl">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open URL</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset theme="system-software-update">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Check for updates</string>
   </property>
   <property name="toolTip">
    <string>Check for updates...</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset theme="media-playback-stop">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="go-top">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Always on top</string>
   </property>
  </action>
  <action name="actionTray">
   <property name="icon">
    <iconset theme="go-bottom">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Hide to tray</string>
   </property>
  </action>
  <action name="actionLite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="preferences-system-windows">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Simple Mode</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="media-skip-forward">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset theme="media-skip-backward">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="audio-volume-muted">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
  </action>
  <action name="actionVolumeUp">
   <property name="icon">
    <iconset theme="audio-volume-high">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Volume up</string>
   </property>
  </action>
  <action name="actionVolumeDown">
   <property name="icon">
    <iconset theme="audio-volume-low">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Volume down</string>
   </property>
  </action>
  <action name="actionRecorder">
   <property name="icon">
    <iconset theme="media-record">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Recorder</string>
   </property>
  </action>
  <action name="actionRecordQuick">
   <property name="icon">
    <iconset theme="media-record">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Quick record</string>
   </property>
   <property name="toolTip">
    <string>Quick record</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="../../tano/resources/misc/misc.qrc">
     <normaloff>:/misc/Qt-logo.svg</normaloff>:/misc/Qt-logo.svg</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="media-playback-stop">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Stop recording</string>
   </property>
   <property name="toolTip">
    <string>Stop recording</string>
   </property>
  </action>
  <action name="actionSchedule">
   <property name="icon">
    <iconset theme="x-office-calendar">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Schedule</string>
   </property>
  </action>
  <action name="actionTeletext">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="text-x-generic-template">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Teletext</string>
   </property>
  </action>
  <action name="actionRecordNow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="media-record">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Instant recording</string>
   </property>
  </action>
  <action name="actionRecordTimer">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Add timer</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="icon">
    <iconset theme="help-browser">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Support</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="image-x-generic">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Take snapshot</string>
   </property>
  </action>
  <action name="actionPreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Preview channels</string>
   </property>
  </action>
  <action name="actionScheduleCurrent">
   <property name="icon">
    <iconset theme="x-office-calendar">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Current show info</string>
   </property>
  </action>
  <action name="actionEditChannels">
   <property name="text">
    <string>Edit channels list</string>
   </property>
   <property name="toolTip">
    <string>Edit channels list</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FancyTabWidget</class>
   <extends>QWidget</extends>
   <header>style/FancyTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../tano/resources/logo/logo.qrc"/>
  <include location="../../tano/resources/misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
